### DISCUSSION TITLE
newbie mac install

### DISCUSSION THREAD

**Discussion by ives31** [Shabinder/SpotiFlyer#1786] [Q&A]
hi
can anyone give me a few pointers on how to install the .jar file on a mac?
Running 12.6 Monteray

many thanks!!

**Comment by SawkeeReemo** [comment]
Same question. The only thing I've been able to figure out is that you somehow have to run this using Java, which Apple abandoned years ago. I tried downloading Java's JRE, but when I try to open Spotifly with it, nothing happens. Would love to know how this is supposed to work. MacOS 10.15.7 Catalina here.

**Comment by danielnbalasoiu** [comment]
Running the failed on my side also, but you can install the app by downloading the [DMG file](https://github.com/Shabinder/SpotiFlyer/releases/download/v3.6.3/SpotiFlyer-3.6.3.dmg) from [releases page](https://github.com/Shabinder/SpotiFlyer/releases).

**Comment by SawkeeReemo** [comment]
Oh, well the main page says: 

> MacOs DMG is not notarized and signed using a certificate , so Use jar in mac for now.

And when I tried to launch it just now, I get this error:
![Screen Shot 2023-01-25 at 9 51 00 PM](https://user-images.githubusercontent.com/60272670/214767393-2c66ff31-b1d6-40de-9a60-d6e66ee111d9.png)

**Reply by lmenti** [reply]
Open Terminal (in folder: Utilities)
sudo spctl --master-disable && xattr -cr /Applications/Spotiflyer.app
(ignore errors)
downlaod Java: https://www.java.com/it/download/
Enjoy!
